docs/source/_templates/sdk_submit_tutorial.rst

```diff
@@ -0,0 +1,259 @@
+{% set short_lang = 'python' %}
+{% if language == 'JavaScript' %}
+    {% set short_lang = 'js' %}
+{% endif -%}
+
+************************************
+Building and Submitting Transactions
+************************************
+
+The process of encoding information to be submitted a distributed ledger is
+generally non-trivial. A series of cryptographic safeguards are used to confirm
+identity and data validity. Sawtooth is no different, but the {{ language }}
+SDK does provide client functionality that abstracts away most of these details,
+and greatly simplifies the process of making changes to the blockchain. If you
+are interested in getting into these details, you should check out the
+:doc:`txn_submit_tutorial_{{ short_lang }}` instead.
+
+
+Creating a Private Key
+======================
+
+In order to confirm your identity and sign the information you send to the
+validator, you will need a 256-bit key. Sawtooth uses the secp256k1 ECSDA
+standard for signing, which means that almost any set of 32 bytes is a valid
+key, and it is fairly simple to generate this using the SDK's *signer* module.
+
+{% if language == 'js' %}
+
+.. code-block:: javascript
+
+    const {signer} = require('sawtooth-sdk')
+
+    const privateKey = signer.makePrivateKey()
+
+{% else %}
+{# Python is the default #}
+
+{% endif %}
+
+.. note::
+
+   This key is the **only** way to prove your identity on the blockchain. Any
+   person possessing it will be able to sign Transactions using your identity,
+   and there is no way to recover it if lost. It is very important that any
+   private key is kept secret and secure.
+
+
+{% include 'partials/encoding_your_payload.rst' %}
+
+.. note::
+
+   This process can be augmented somewhat by using the *payload encoder* feature
```

Comment:
Suggestion: rewrite to be more clear: "augmented somewhat."  What does this mean, and how can it be augmented? Maybe I'm just dense. ;)